packages/icalendar/tests/Recur.pkl-expected.pcf

```diff
@@ -0,0 +1,34 @@
+examples {
+  ["with date"] {
+    """
+    DTSTART;VALUE=DATE:20000101
+    RRULE:FREQ=YEARLY;INTERVAL=1;UNTIL=20250101;BYDAY=MO,-1FR;BYMONTHDAY=15,-15;BYYEARDAY=100,-100;BYMONTH=1,11;WKST=MO;BYSETPOS=-1,1
+    """
+    "Expected value of type `* DateTime(utc.implies(tzid == null))   |Date(tzid == null)`, but got a different `icalendar.Date`. Value: new ModuleClass { y = ?; m = ?; d = ? }"
```

Comment:
Error messages shouldn't be part of an API (if/when we improve error messages in pkl-core, we don't want our packages to break). Can you change these tests to something like:

```pkl
    module.catchOrNull(() -> new Recur {
      dtstart = DateTime.DateTime("20000101T000000")
      freq = "YEARLY"
      until = DateTime.DateTime("20250101T000000Z")
    }.toString()) != null
```